C/Areas of Different Geometrical Figures

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
This program  is perfect just add the extension of this file

Reply:
Everything has been perfect! Just add the extension to this file and your PR is merged!

Reply:
Ok Nice..Also check out c++ programs...